internal/k3s/install.go

```diff
@@ -0,0 +1,238 @@
+package k3s
+
+import (
+	"bufio"
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"io/fs"
+	"os"
+	"os/exec"
+	"path"
+	"regexp"
+	"strings"
+
+	"github.com/rs/zerolog"
+	"github.com/weka/gohomecli/internal/bundle"
+	"github.com/weka/gohomecli/internal/utils"
+)
+
+const k3sImagesPath = "/var/lib/rancher/k3s/agent/images/"
+const defaultLocalStoragePath = "/opt/local-path-provisioner"
+
+var (
+	ErrExists = errors.New("k3s already installed")
+)
+
+var (
+	k3sBundleRegexp = regexp.MustCompile(`k3s.*\.(tar(\.gz)?)|(tgz)`)
```

Comment:
minor: I think we may just hardcode that because we are the ones who will create a package :)